openspec/specs/upgrade-command/spec.md

```
@@ -0,0 +1,36 @@
+# upgrade-command Specification
+
+## Purpose
+
+TBD - created by archiving change upgrade-01-install-method-aware. Update Purpose after archive.
+## Requirements
+### Requirement: Upgrade command must respect installation method
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)